telearm/input/gamepad_reader.py

```
@@ -0,0 +1,255 @@
+        if not self.use_evdev and PYGAME_AVAILABLE:
+            if not self.pygame_initialized:
+                pygame.init()
+                pygame.joystick.init()
+                self.pygame_initialized = True
+            
+            if pygame.joystick.get_count() > 0:
+                self.device = pygame.joystick.Joystick(0)
+                self.device.init()
+                print(f"Using pygame gamepad: {self.device.get_name()}")
+            else:
+                raise RuntimeError("No gamepad detected")
```

Ruff (0.14.3)

Avoid specifying long messages outside the exception class

(TRY003)